video-scripts/short_introduction/oft_short_introduction.yaml

```
@@ -0,0 +1,147 @@
+  - !Wait 2s
+  - !Clear
+
+  - !Marker User Requirements
+  - !Command
+    command: "# Let's look at a minimal requirement specification file written in Markdown."
+  - !Wait 2s
+  - !Command
+    command: show user_requirements.md
+  - !Wait 2s
+  - !Command
+    command: "# In line 1 we have a unique requirement ID."
+  - !Wait 2s
+  - !Command
+    command: "# Then there is the description."
+  - !Wait 2s
+  - !Command
+    command: "# And finally, in line 4 we ask for coverage in a design."
+  - !Wait 5s
+  - Clear
+
+  - !Command
+    command: "# We will now run a trace over that file and it must be incomplete, since the design is missing."
+  - !Wait 2s
+  - !Command
+    command: oft trace user_requirements.md
+  - !Wait 2s
+  - !Command
+    command: "# Sure enough, OFT tells us that the coverage by a design item is missing."
+  - !Wait 5s
+  - !Clear
+    
+  - !Marker Design
+  - !Command
+    command: "# Now let's look at a matching design."
+  - !Wait 2s
+  - !Command
+    command: show design.md
+  - !Wait 2s
+  - !Command
+    command: "# As you can see, the structure is the same as in the user requirement."
+  - !Wait 2s
+  - !Command
+    command: "# What's new is that in line 5 we declare that this design item covers our user requirement."
+  - !Wait 5s
+  - !Clear
+
+  - !Command
+    command: "# If we trace both files together, the covering code is missing, so we get another error."
+  - !Wait 2s
+  - !Command
+    command: oft trace user_requirements.md design.md
+  - !Wait 2s
+  - !Command
+    command: "# Note that the error is transitive."
+  - !Wait 2s
+  - !Command
+    command: "# The design item is not covered, so the corresponding user requirement has only shallow coverage."
+  - !Wait 5s
+  - !Clear
+
+  - !Command
+    command: "# But we can verify that the design (\"dsn\") covers the requirement (\"req\"), ignoring the required code for now."
+  - !Wait 2s
+  - !Command
+    command: oft trace user_requirements.md design.md -a req,dsn
+  - !Wait 2s
+  - !Command
+    command: "# As you can see, the design covers the user requirement correctly."
+  - !Wait 2s
+  - !Command
+    command: "# This allows checking the consistency of the requirement chain step-by-step."
+  - !Wait 5s
+  - !Clear
+
+  - !Marker Tracing Into Code
+  - !Command
+    command: "# The last piece of the puzzle is a tag in the code that tells us where the implementation for the design is."
+  - !Wait 2s
+  - !Command
+    command: show square_root.py
+  - !Wait 2s
+  - !Command
+    command: "# Note the special comment in line 4 of the Python script: [impl->...]"
+  - !Wait 2s
+  - !Command
+    command: "# If we trace all three files together, the requirement chain is now complete."
+  - !Wait 2s
+  - !Command
+    command: oft trace user_requirements.md design.md square_root.py
+  - !Wait 5s
+  - Clear
+
+  - !Marker Summary
+  - !Command
+    command: '# OpenFastTrace (OFT) lets you...'
```

```suggestion
    command: '# OpenFastTrace lets you...'
```
Either introduce abbreviation in the beginning or better omit it completely.